python/tvm/relay/frontend/tflite.py

```diff
@@ -199,6 +212,10 @@ def get_tensor_type_str(self, tensor_type):
         raise NotImplementedError("Tensor type {} is currently not supported"
                                   .format(str(tensor_type)))
 
+    def has_same_qnn_params(self, tensor_a, tensor_b):
+        return tensor_a.qnn_params['scale'] == tensor_b.qnn_params['scale'] and \
+                tensor_a.qnn_params['zero_point'] == tensor_b.qnn_params['zero_point']
```

Comment:
Minor code style suggestion. `tensor_a` be `lhs_tensor` and `tensor_b` be `rhs_tensor` like we did in `convert_elemwise`.